src/app/ui/components/link/link.tsx

```diff
@@ -0,0 +1,21 @@
+import { styled } from 'leather-styles/jsx';
```

Comment:
@fbwoolf apologies for the late feedback here, but I wonder if we should provide a sensible default for `<Link`? 

Mentioning as I noticed that some link text is not visible where we don't explicitly set it. 

https://github.com/leather-wallet/extension/assets/2938440/3bc815df-dd9f-472b-8b68-4bc6adb14b06


See here on `sign-in` page for example. I can fix this in my PR by specifying variant, but I wonder if we need to audit all updated `<Link` components, or else default them to have variant `text` if not provided?

Reply:
There is a `defaultVariant` in the link recipe. Feel free to change it if something else works better. Rn it defaults to `size="md"` and `variant="underlined"`, which I think makes sense, no?

Reply:
I think this is more a color thing with light/dark mode bc it is supposed to be the default underlined link and is ok in dark mode?

![Screenshot 2024-01-31 at 12 18 00 PM](https://github.com/leather-wallet/extension/assets/6493321/5fc243ff-4b1b-48ac-bbfb-e368fe20aa75)

Reply:
Fixed here... https://github.com/leather-wallet/extension/pull/4885

![Screenshot 2024-01-31 at 12 39 20 PM](https://github.com/leather-wallet/extension/assets/6493321/3337590f-d6a4-40ae-bf9e-1e8db53baab7)

Reply:
Thanks @fbwoolf 👏 that default does make sense. I seeing this a lot on my containers work but it's looking a lot better since rebasing this. 

I noticed another issue, unrelated where the `disabled` button blends into the background. 
![Screenshot 2024-02-01 at 05 52 13](https://github.com/leather-wallet/extension/assets/2938440/93072770-205f-445d-b75a-53ce2e349b28)

I will check that out, just mentioning before I forget. We may need a different design for disabled on the same BG colour

Reply:
Just small observation that this button should be contained in a white footer.

Reply:
@mica000 I have been just working on this and changed the footer to be transparent, like the header. 

Do you want it to always be white? It looks kind of odd with the off white background

https://github.com/leather-wallet/extension/assets/2938440/84eac49c-c611-4924-969d-5d5f6f97a22d

Reply:
OK, thanks for the confirmation @mica000. In my screenshot the height was incorrect there, it will adjust based on the content. 

 I'll keep the footer white and add the border which makes a difference. I think it will be tricky to only show the top border if the above content has a scroll so I will just add a border in all cases for now then try and solve that later, is that OK? cc @fabric-8 

Here it seems we have more changes to make too, for example in `PSBT` should inputs and outputs have a white background also? The same as above in `From account...` ? 

I think we should make a `card` component for those things. Do we have a base component for that? It's not urgent now but we can add it. 


![Screenshot 2024-02-02 at 05 10 31](https://github.com/leather-wallet/extension/assets/2938440/e880cb09-27c1-4938-a076-add661373264)

Reply:
I do agree but also cry a little bit deep inside 🥲

Reply:
> Here it seems we have more changes to make too, for example in PSBT should inputs and outputs have a white background also? The same as above in From account... ?

Ah, so dont worry, this is part of the new designs for approval UX - but its a good topic to discuss on how we want to proceed with merging new designs with old designs as some areas of the old are being affected.

A fix for this could be adding a white background to the current cards in PSBT, unitl we build and release the new ones.

WDYT?

Reply:
Sounds good but I won't do that as part of #4370 . When we are testing that we can check all the screens and make a list of where we need to add a white background and do it as a separate task